### DISCUSSION TITLE
heroku-24 stack with new heroku google chrome buildpack

### DISCUSSION THREAD

**Discussion by davidwessman** [rubycdp/ferrum#481] [General]
Heroku has deprecated its old buildpack for Google Chrome with a new one:
https://blog.heroku.com/improved-browser-testing-on-heroku-with-chrome

When using the new buildpack and heroku-24 I ran into issues with:

```
No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/main/docs/linux/suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
```

I am not quite sure which part has changed between heroku-22 (ubuntu 22 based) vs the buildpack.
I will make test app on Heroku using heroku-22 and the new buildpack to see if I get the same error.

Have anyone else ran into similar issues? The only reference to the sandbox is in 
https://github.com/rubycdp/ferrum?tab=readme-ov-file#docker

**Comment by route** [comment]
That's right you should pass `no-sandbox` flag when running Chrome, just add `browser_options`

**Comment by davidwessman** [comment]
The old google chrome build pack sets sandbox by default. So adding the option is the way to go
